### PR TITLE
Playback Analytics & Notification Icons

### PR DESCRIPTION
**Pull request by sam33rdhakal**
updates
* Adds audio completed, interrupted analytics events
* Adds notification icons

### CODE REVIEW

File: db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package edu.artic.db.models
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+
+fun ArticObject.getAudio(): ArticAudioFile? {
+    return if (this.audioCommentary.isNotEmpty()) {
+        this.audioCommentary.first().audioFile
```
Comment (agrosner) [inline db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt]: we could cleanup this method to:
```
fun ArticObject.getAudio(): ArticAudioFile? = this.audioCommentary.firstOrNull()?.audioFile
```

Comment (agrosner) [reply db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt]: also renaming it to an Extension property might be better:
```
val ArticObject.audioFile: ArticAudioFile? 
  get() = this.audioCommentary.firstOrNull()?.audioFile
```
And more semantic to Kotlin, not Java "getters".

File: db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package edu.artic.db.models
```
Comment (agrosner) [inline db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt]: Rename file (too generic) or put it in the same file as `ArticObject` as an extension method residing near the class.

File: media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt [hunk 3]
```diff
@@ -144,16 +150,24 @@ class AudioPlayerService : Service() {
 
     val disposeBag = DisposeBag()
 
+    @Inject
+    lateinit var analyticsTracker: AnalyticsTracker
+
     override fun onCreate() {
+        AndroidInjection.inject(this)
```
Comment (agrosner) [inline media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt]: There is a `DaggerService` that does this for you, if you prefer.

Comment (sam33rdhakal) [reply media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt]: Updated.

File: db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt (hunk 2 above)
Comment (sam33rdhakal) [reply db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt]: Removed this file and moved the extension method to ArticObject as the extension property.

File: db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt (hunk 1 above)
Comment (sam33rdhakal) [reply db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt]: I implemented extension property.

**Pull request merged**